test/spec/stores/users.test.js

```diff
@@ -27,31 +28,32 @@ describe('UsersStore', () => {
         }
         actions.fetch();
       });
-    };
+    }
+
     store.listen(handleChange);
     actions.fetch();
   });
 
   it('should merge users when on new fetch', function(done) {
     // add dummy user in store
-    actions.fetchBySeedSuccess({name: 'foo', seed: 'bar'});
+    actions.fetchBySeedSuccess({ name: 'foo', seed: 'bar' });
 
-    const handleChange = function({users}) {
+    function handleChange({ users }) {
```

Comment:
'destructuring expression' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).